yaksh/copy_delete_files.py

```diff
@@ -0,0 +1,29 @@
+import shutil
+import os
+import zipfile
+
+
+class CopyDeleteFiles(object):
+
+    def copy_files(self, file_paths):
```

Comment:
Why are these methods of a class?  No need for an object when you don't need one.  Just make them functions and also change the module name to utils or something like that.  Why isn't there a docstring, especially when file_paths is expected to be a specific form of data?